src/redux/products/products.sagas.js

```diff
@@ -1,9 +1,131 @@
 import { takeEvery, call, put } from 'redux-saga/effects';
 import { push } from 'connected-react-router';
-import { setAllProducts, setLoading } from './products.actions';
+import {
+  setAllProducts,
+  setLoading,
+  setAllFilterProducts,
+  setPagesCount
+} from './products.actions';
 import { setError } from '../error/error.actions';
 import getItems from '../../utils/client';
-import { GET_ALL_PRODUCTS } from './products.types';
+import { GET_ALL_PRODUCTS, GET_FILTRED_PRODUCTS } from './products.types';
+
+export function* handleFilterLoad({
+  payload = {
```

Comment:
I think it is better to take this value from the redux store here using `yield select`, it will definitely simplify your action

Reply:
fixed